### PR TITLE
changing towards new naming convention for config csvs

### PR DESCRIPTION
**Pull request by kavyasreedhar**
Joey has simpler names for the config csvs (example: https://github.com/dillonhuff/clockwork/tree/lower_ubuffer/aha_garnet_design/cascade/lake_collateral/ub_conv1_stencil_op_hcompute_conv1_stencil_1_16_to_conv1_stencil_op_hcompute_conv2_stencil_1_11), so this puts the default for the Lake wrapper generation as having those names instead of "input" or "output" prepended to the filename. The compiler team is still in the process of transitioning to this new naming convention, so we keep the in_file_name and out_file_name args for now for backwards compatibility (mainly for the CI conv_33 test to pass), but we should be able to delete them in the near future as config files are updated and streams are generated.

Also reorganizes testbench helper functions so Lake wrapper functions for test calls can be installed with Lake and used elsewhere for wrapper file + garnet tests so code for Clockwork paths / args for tb is not duplicated between garnet and lake requiring changes to be updated in both places

Also checks if config files exist to enable compiler to configure a stencil valid only UB tile.

Garnet PR: https://github.com/StanfordAHA/garnet/pull/713
aha branch: csv_names

### CODE REVIEW

**Review by mbstrange2 — APPROVED** [review]

File: tests/wrapper_lake.py [hunk 1]
```diff
@@ -4,14 +4,14 @@
 
 from lake.utils.util import generate_lake_config_wrapper
 from lake.utils.util import check_env
-from test_lake import base_lake_tester
+from lake.helper_test import base_lake_tester
```
Comment (mbstrange2) [inline tests/wrapper_lake.py:7]: this will break right?

File: lake/utils/test_infra.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+    config_path = lc + "conv_3_3_recipe/buf_inst_input_10_to_buf_inst_output_3_ubuf"
+    stream_path = ls + "conv_3_3_recipe/buf_inst_input_10_to_buf_inst_output_3_ubuf_0_top_SMT.csv"
+    in_file_name = "input_"
+    out_file_name = "output_2_"
+    return config_path, stream_path, in_file_name, out_file_name
+
+
+# not all of these tests/folders still exist,
+# update in_file_name and out_file_name args if needed
+def test_gaussian():
```
Comment (mbstrange2) [inline lake/utils/test_infra.py:105]: can we just have the actual tests in tests/test_lake.py? The testing infrastructure can be here.

**Review by mbstrange2 — CHANGES_REQUESTED** [review]
Seems one of the imports for wrapper_lake.py is out of date, if we can have test_lake have the actual tests in them I think that makes the most sense. Or to make a test base class.

File: lake/utils/test_infra.py (hunk 2 above)
Comment (kavyasreedhar) [reply lake/utils/test_infra.py:105]: these tests are all out of date and the paths don't exist for all of them,...would you be okay if I just delete them?

There is one test function in tests/test_lake.py now and that just takes the args (csv paths) for pytest and tests the actual tests there (right now, just conv_3_3).

File: tests/wrapper_lake.py (hunk 1 above)
Comment (kavyasreedhar) [reply tests/wrapper_lake.py:7]: yes, thanks for catching that! I'll update that.

File: lake/utils/test_infra.py (hunk 2 above)
Comment (kavyasreedhar) [reply lake/utils/test_infra.py]: I deleted old tests to avoid confusion, lmk if you want that reverted.

**Review by mbstrange2 — APPROVED** [review]
LGTM

**Review by mbstrange2 — APPROVED** [review]
LGTM

**Pull request merged**